src/checkers/ast.rs

```diff
@@ -754,6 +763,18 @@ where
                 decorator_list,
                 body,
             } => {
+                if self.settings.rules.enabled(&Rule::ModelStringFieldNullable) {
+                    let mut errors =
+                        flake8_django::rules::ModelStringFieldNullable::check(bases, body);
+                    self.diagnostics.append(&mut errors);
```

Comment:
We'd generally write this as: `self.diagnostics.extend(flake8_django::rules::ModelStringFieldNullable::check(bases, body))`.